### PR TITLE
feat: skip render author if they are not object

### PR DESCRIPTION
**Pull request by niktverd**

### CODE REVIEW

**Review by Feverqwe — APPROVED** [review]

File: src/components/DocPage/DocPage.tsx [hunk 1]
```diff
@@ -376,7 +376,7 @@ class DocPage extends React.Component<DocPageInnerProps, DocPageState> {
     private renderAuthor(onlyAuthor: boolean) {
         const {meta, lang} = this.props;
 
-        if (!meta?.author) {
+        if (!meta?.author || typeof meta?.author !== 'object') {
```
Comment (Lunatic174) [inline src/components/DocPage/DocPage.tsx:379]: According to typings, it's either object or undefined, it's not true?

https://github.com/yandex-cloud/docs-components/blob/518599d8c025ed9a2437b3330751fddd88485cd4/src/models/index.ts#L86

https://github.com/yandex-cloud/docs-components/blob/518599d8c025ed9a2437b3330751fddd88485cd4/src/models/index.ts#L122

Comment (Lunatic174) [reply src/components/DocPage/DocPage.tsx:379]: Sounds strange for me...

Comment (Feverqwe) [reply src/components/DocPage/DocPage.tsx:379]: I think need add unknown type, cause metadata may contain any data.

**Review by Feverqwe — APPROVED** [review]

File: src/utils/index.ts [hunk 2]
```diff
@@ -95,3 +95,13 @@ export function isInvalidEmail(email: string) {
 
     return !EMAIL_RE.test(email);
 }
+
+export function isContributor(contributor: unknown): contributor is Contributor {
+    if (!contributor) {
+        return false;
+    }
```
Comment (Lunatic174) [inline src/utils/index.ts:102]: I suppose it's better to add check for object here:

```ts
    if (!contributor || typeof contributor !== 'object') {
        return false;
    }
```

File: src/utils/index.ts [hunk 3]
```diff
@@ -95,3 +95,13 @@ export function isInvalidEmail(email: string) {
...
+
+    const {login, name, email} = contributor as Contributor;
+
+    return Boolean(login || name || email);
```
Comment (Lunatic174) [inline src/utils/index.ts:106]: if all of them are empty - isn't it an empty Contributor type?

**Review by Lunatic174 — CHANGES_REQUESTED** [review]

File: src/utils/index.ts (hunk 3 above)
Comment (niktverd) [reply src/utils/index.ts]: changed it on `in` clause

File: src/utils/index.ts (hunk 2 above)
Comment (niktverd) [reply src/utils/index.ts]: fixed

**Review by Lunatic174 — APPROVED** [review]

**Pull request merged**